app/src/main/res/values-en/strings.xml

```diff
@@ -167,7 +167,7 @@
   <string name="settings_depositGuarantees_title">Deposit Guarantees</string>
   <string name="settings_depositGuarantees_subtitle">Set your default guaranteed minimum for estimated deposits</string>
   <string name="settings_depositGuarantees_text">Set the guaranteed minimum deposit to be applied whenever a deposit in a transaction can only be estimated.
-
+\n\n
```

Comment:
Are these changes intended?

Reply:
no, I think we have gradle task that adds those on every build?